solvcon/solver_legacy.py

```diff
@@ -286,6 +288,10 @@ def bind(self):
         for it in range(self.ncore):
             self.arglists.append([byref(self.exd), c_int(0), c_int(0)])
 
+    def check_blk_pointer(self):
```

Comment:
I suggest to rename it to simply check_pointer**s**(), so that it matches the name of the underneath helper.

Also note this legacy Solver class uses ctypes to interface C.  I am not sure if this matters or not.